offers.md

```diff
@@ -51,34 +51,33 @@ curl -X POST https://api.fidel.uk/v1/offers \
-* `funded: type`: Type of Offer funding. Possible values are `"merchant"`, `"card-linking"` and `"affiliate"`. In the test environment, you can only create card-linked Offers, so the funding type will always be `"card-linking"`.
-* `maxTransactionAmount`: Maximum amount for a Transaction to qualify for the Offer. For example, suppose your promotion sounded something similar to "Save 25% on purchases over £50, save 40% on purchases over £100". In that case, the first Offer should have a `maxTransactionAmout` of £100.
-* `minTransactionAmount`: Minimum amount for a Transaction to qualify for the Offer. For example, suppose your promotion sounded something similar to "Save 25% on purchases over £50". In that case, the Offer should have a `minTransactionAmount` of £50.
-* `metadata`: Object with your own metadata, will be returned on the Offer object.
-* `returnPeriod`: Number of days between when a Transaction was created and when a Transaction qualifies for the Offer. The qualified Transaction will have the `offer.qualificationDate` set to the creation date plus the number of days in the return period.
-* `schemes`: Array of schemes for which a Transaction qualifies for the Offer. Possible values are `"amex"`, `"mastercard"` and `"visa"`.
-* `type: maxRewardAmount`: Numeric value of the maximum amount to be awarded for the Offer. Only applies to `discount` type Offers.
-
+- `activation`: An object, showing if the Offer needs activation or not. Default is `{ enabled: false }`.
+- `activation: enabled`: Boolean showing if the Offer needs to be activated on Cards or not. If it's `true`, the `activation` object also has a `qualifiedTransactionsLimit` property. **Please read the section below on Offer Activation before using this parameter.**
+- `activation: qualifiedTransactionsLimit`: Number of Transactions to qualify for each Offer activation. Default is 1.
+- `daysOfWeek`: Array of numbers, with possible values from 0 to 6, to indicate the days of the week. 0 = Sunday, 1 = Monday, etc.
+- `endDate`: The date to automatically end the Offer. Same as `startDate`, the time will be a local time relative to the Location where the Offer was active.
+- `funded: id`: Unique identifier for the account that funds the Offer. For self-funded Offers, this is not required. In the test environment, all Offers are self-funded, so this will always be the same as your `accountId`.
+- `funded: type`: Type of Offer funding. Possible values are `"merchant"`, `"card-linking"` and `"affiliate"`. In the test environment, you can only create card-linked Offers, so the funding type will always be `"card-linking"`.
+- `maxTransactionAmount`: Maximum transaction amount generating a proportional reward. Example, 10% reward with max. spend at 100$ can’t generate a reward larger than 10$.
```

Comment:
Maybe let's keep the £ instead of $ since we probably are using always £.

Reply:
Consider something like this on the example.
`Example, 10% reward with max transaction amount of £100 can't generate a reward larger than £10, even if the transaction amount is higher than £100.

Reply:
Nice suggestion regarding consistency with the currency.

Regarding the example, I would write "For example, ..."

Please use ' instead of ’ ("can't").

Reply:
that example is part of the content we have in the dashboard, should we enforce that consistency across our ecosystem communication?

Reply:
hmm good point 🤔